README.md

```diff
@@ -43,9 +43,9 @@ printer.configure_options(PrinterConfig(delimiter_left="|", delimiter_right="|",
 The `Printer` class also has a `print()` method that can be used to print text. The `print()` method can be used as follows:
 
 ```python
-from fancyprint import Printer
-from fancyprint.helpers.types import PrintConfig
-from fancyprint.helpers.constants import Align
+from src.fancyprint import Printer
+from src.fancyprint.helpers.types import PrintConfig
+from src.fancyprint.helpers.constants import Align
 printer=Printer()
```

Comment:
It's still a package, remove the src. We don't need it.

Reply:
ok